payjoin/src/send/mod.rs

```diff
@@ -11,7 +11,7 @@ use std::str::FromStr;
 use bitcoin::psbt::Psbt;
 use bitcoin::{Amount, FeeRate, Script, ScriptBuf, TxOut, Weight};
 pub use error::{BuildSenderError, ResponseError, ValidationError};
-pub(crate) use error::{InternalBuildSenderError, InternalProposalError, InternalValidationError};
+pub(crate) use error::{InternalBuildSenderError, InternalProposalError};
```

Comment:
was this `InternalValidationError` used in this file originally? I I havev't tested if I got a unused code warning on master but I don't see where i deleted anything that used this here

Reply:
Ok nvm I see what I did